README.md

```diff
@@ -1,24 +1,149 @@
+## 🛠 Built With <a name="built-with"></a>
+- Rails
+- PostgreSQL
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- [ ] **[creating a data model]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+- Laptop
+- Github
+- Git
+- Ruby
+- Rails
+- Linters
+
+### Prerequisites
+
+In order to run this project you need:
+- To install Ruby following these steps:
+- Install the latest version on ruby+devkit [rubyinstaller](https://rubyinstaller.org/downloads/)
+- Follow the installation steps [here](https://stackify.com/install-ruby-on-windows-everything-you-need-to-get-going/)
+- install the [linters for Ruby](https://github.com/microverseinc/linters-config/tree/master/ruby)
+
+### Setup
+
+Clone this repository to your desired folder:
+Example commands:
+```sh
+  cd my-folder
+  git https://github.com/Bettenoch/creativeblog.git
+```
+### Install
+Install this project with:
+Example command:
+```sh
+  cd vortex
+  gem install
+```
+### Usage
+To run the project, execute the following command:
+Example command:
+```sh
+  rails server
+  OR
+  rails s
+```
+### Run tests
+To run tests, run the following command:
+Example command:
+```sh
+  bin/rails test test/models/post_test.rb
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Bett Enoch**
+
+- GitHub: [@bettenoch](https://github.com/Bettenoch)
+- Twitter: [@BybEnoch](https://twitter.com/BybEnoch)
+- LinkedIn: [@GeoIntelligence](https://www.linkedin.com/in/bett-kipngeno-enock-8b5153214/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[creating a data model]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/kkmanuu/Blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give it a ⭐️ and follow me on github.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse Team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## :memo: License <a name="license"></a>
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] KIndly fix the link to your MIT License page it is currently broken, when I click  on the link it shows a  `404 - page not found`